python/mxnet/rnn/rnn_cell.py

```diff
@@ -788,6 +788,12 @@ def unpack_weights(self, args):
     def pack_weights(self, args):
         return _cells_pack_weights(self._cells, args)
 
+    def reset(self):
+        super(SequentialRNNCell, self).reset()
+        if hasattr(self, '_cells'):
```

Comment:
You are right. It is better to put this in the BaseRNNCell. I will work for gluon too. Thanks for your suggestions. @szha